message/message_test.go

```diff
@@ -26,15 +26,13 @@ func TestNewWithEmptyValue(t *testing.T) {
 // message.New should return an error if no topic name is provided.
 func TestNewWithEmptyTopic(t *testing.T) {
 	_, err := message.New("", "Foo")
-	require.Error(t, err)
-	require.Equal(t, "messages require a non-empty topic", err.Error())
+	require.EqualError(t, err, "messages require a non-empty topic")
```

Comment:
Nice!

Reply:
:tulip:  I've actually been thinking that we should define a `FeeliceError` type that supports the error interface, but which can have an ID which matches a constant - that would make this kind of assertion cleaner, and likewise in client code.  Testing what kind of error you're seeing is something go programs often do badly.

Reply:
Something like that: https://github.com/heetch/regula/blob/master/errors.go?